distributed/scheduler.py

```diff
@@ -86,13 +86,18 @@
 from .protocol.highlevelgraph import highlevelgraph_unpack
 
 try:
-    from cython import double, Py_ssize_t
+    from cython import cclass, double, Py_hash_t, Py_ssize_t
 except ImportError:
     from ctypes import (
         c_double as double,
+        c_ssize_t as Py_hash_t,
```

Comment:
For context, this is reasonable as [Python does the same thing itself]( https://github.com/python/cpython/blob/b3b98082c5431e77c64cab2c85525a804436b505/Include/pyport.h#L114 ). Though it shouldn't matter whether we get this right in the pure Python case as this doesn't have any effect on the implementation besides documentation. Just noting this to provide background as to how this was decided.